benchmark/crypto/aes-gcm-throughput.js

```diff
@@ -8,10 +8,10 @@ var bench = common.createBenchmark(main, {
 });
 
 function main(conf) {
-  var message = (new Buffer(conf.len)).fill('b');
+  var message = (Buffer.allocUnsafe(conf.len)).fill('b');
```

Comment:
This is outside the bench, so `Buffer.alloc(conf.len, 'b')` would not change the result, but is more clean.